crates/biome_js_analyze/tests/specs/nursery/noJsxPropsBind/valid.jsx

```diff
@@ -2,27 +2,29 @@
 
 import { useCallback } from "react";
 
+function onClick() {}
+
 function Foo() {
-	function onClick() { }
-	return <Bar onClick={onClick} />
+	return <Bar onClick={onClick} />;
 }
 
+const onClick = function () {};
+
 function Foo() {
-	const onClick = function () { }
-	return <Bar onClick={onClick} />
+	return <Bar onClick={onClick} />;
 }
 
+const onClick = () => {};
+
 function Foo() {
-	const onClick = () => { }
-	<Bar onClick={onClick} />
+	<Bar onClick={onClick} />;
 }
 
 <>
-	<Foo onClick={this.handleClick}></Foo>;
-	<Foo onClick={bind()}></Foo>;
-</>
+	<Foo onClick={this.handleClick}></Foo>;<Foo onClick={bind()}></Foo>;
+</>;
```

Comment:
This seems a bit weirdly formatted

Reply:
Okay. I want to get this done, but I'm also on a completely new machine and haven't been able to get codegen to run. If you could do this once everything else looks good to you, I'd appreciate it. :)